source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -95,11 +95,16 @@ Version compatibility
 
 We validated these steps against the following software package versions:
 
-* |CL| 26240 (Lower version not supported)
+* |CL| 26240 (Minimum supported version)
```

Comment:
Are these components all >= minimum requirements, or very specifically those versions only?
I understand the point of this is the *tested* version, but is the intent that newer version should also work?

Reply:
Newer versions should work.